### PR TITLE
updating link to slides

### LINKED ISSUE #45: ["Problem"]
slide link needs to be updated on index

### PR DESCRIPTION
**Pull request by carriewright11**
Resolves #45  to make the link to the slides the correct link

### CODE REVIEW

**Comment by carriewright11** [comment]
@cansavvy  this broken url is confusing me... I think it has to do with having parentheses in the url again... maybe this repo just needs updating?

**Comment by cansavvy** [comment]
https://github.com/jhudsl/OTTR_Template/issues/631

File: 02-Data_Privacy.Rmd [hunk 1]
```diff
@@ -25,7 +25,7 @@ Note that these are general definitions and whether something counts as PII or P
 
 PII (personal identifiable information) are aspects of a person that could allow you to identify a person.
 
-PII is defined by the [US Department of Labor](https://www.dol.gov/general/ppi) as:
```
Comment (carriewright11) [inline 02-Data_Privacy.Rmd:28]: This is not in the current version

File: index.Rmd [hunk 2]
```diff
@@ -20,4 +20,4 @@ knitr::write_bib(c(
...
...ch Education Resource. This material was created by the ITCR Training Network (ITN)  which is a collaborative effort of researchers around the United States to support cancer informatics and data science training through resources, technology, and events. This initiative is funded by the following grant:  [National Cancer Institute (NCI)](https://www.cancer.gov/)  UE5 CA254170. Our courses feature tools developed by ITCR Investigators and make it easier for principal investigators, scientists, and analysts to integrate cancer informatics into their workflows. Please see our website at [www.itcrtraining.org](www.itcrtraining.org) for more information. Except where otherwise indicated, the contents of this course are available for use under the Creative Commons Attribution 4.0 license. You are free to adapt and share the work, but you must give appropriate credit, provide a link to the license, and indicate if changes were made. Sample attribution: Data Management for Cancer Research by Johns Hopkins Data Science Lab (CC-BY 4.0). You can download the illustrations by clicking [here](https://docs.google.com/presentation/d/1B4LwuvgA6aUopOHEAbES1Agjy7Ex2IpVAoUIoBFbsq0/edit?usp=sharing).
```
Comment (carriewright11) [inline index.Rmd:23]: This is not in the current version

**Pull request merged**